web-ui/src/main/resources/catalog/components/utility/UtilityDirective.js

```diff
@@ -942,6 +942,8 @@
         scope: {
           date: '=gnBootstrapDatepicker',
           dates: '=dateAvailable',
+          dateMin: '=dateMin',
+          dateMax: '=dateMax',
           config: '=config',
```

Comment:
You can probably use the config property directly ? instead of adding 2 new properties - there is lot of other config option in date picker. eg. it is used for facet on date range which select by year for example.

Reply:
Okay, I will adapt the PR with your comment thanks

Reply:
However on sextant 6.7 the config property does not exist

Reply:
That's the issue when we fork and don't apply changes first in GeoNetwork....